### PR TITLE
NGP Test Time Step reduction

### PR DESCRIPTION
**Pull request by PaulMullowney**
The change here is motivated by the amount of time these tests take under the Clang address sanitizer. Right now, each test is running 2-4 hours with asan when doing 10 time steps. This is way too long. I think 3 time steps is sufficient to see that the trajectory is ok.

**Pull-request type:**
- [ ] Bug fix 
- [ ] Documentation update
- [x] Feature enhancement

## Description of the pull-request

*Provide a description of your proposed changes. If there is a related issue, please specify.**

## Checklist

**NOTE** The checklist below is a suggestion and not mandatory. You don't have
to _check all boxes_ to submit a pull request. However, please add a brief
explanation in your pull request summary explaining the omission for the benefit
of reviewers and developers.

*All pull requests*
- [x] Builds successfully (*must test on at least one system/compiler combination*)
  - Operating systems 
    - [x] Linux
    - [ ] MacOS
  - Compilers 
    - [x] GCC
    - [ ] LLVM/Clang
    - [ ] Intel compilers
    - [x] NVIDIA CUDA
- [ ] Compiles without warnings
- [ ] Passes all unit tests
- [ ] Passes all regression tests
- [ ] Documentation builds without errors

*For new code updates*
- [ ] Documentation updates - additions to user, theory, and verification manuals
- [ ] New unit tests providing coverage for new code, bug fixes etc.
- [ ] New regression tests exercising the new code

### CODE REVIEW

**Review by jrood-nrel — APPROVED** [review]
Looks good. I think we will want to wait until we resolve the dashboard and hopefully merge this next week.

**Comment by sayerhs** [comment]
@PaulMullowney We start with a nearly uniform flow and I am a little concerned about not running sufficient timesteps for the GCC and CUDA versions – I do worry that may be even 10 time steps there is no sufficient. . May I recommend an alternate strategy – exclude this tight tolerance test from the clang/ASAN run and instead add a new test with 3 time steps to catch memory errors in ASAN build? This will keep the tighter checks in GCC/CUDA and allow it to catch things bugs in physics that grow slowly.

**Comment by PaulMullowney** [comment]
@sayerhs That's a good idea. I'll rework this PR.

**Comment by jrood-nrel** [comment]
I would prefer not to increase test complexity. I will solve this with more parallelism in the testing so it completes before the morning.

**Comment by PaulMullowney** [comment]
Ok. I’ll rescind this PR

From: Jon Rood ***@***.***>
Reply-To: Exawind/nalu-wind ***@***.***>
Date: Friday, April 30, 2021 at 2:16 PM
To: Exawind/nalu-wind ***@***.***>
Cc: "Mullowney, Paul" ***@***.***>, Mention ***@***.***>
Subject: Re: [Exawind/nalu-wind] NGP Test Time Step reduction (#857)


CAUTION: This email originated from outside of NREL. Do not click links or open attachments unless you recognize the sender and know the content is safe.

I would prefer not to increase test complexity. I will solve this with more parallelism in the testing so it completes before the morning.

**Comment by PaulMullowney** [comment]
Closing this PR. Jrood prefers to use more parallelism to speed up test time.

**Pull request closed without merging**
